### PR TITLE
add package topics for package:shelf_router_generator

### PR DESCRIPTION
**Pull request by devoncarew**
- add package topics for package:shelf_router_generator

### CODE REVIEW

**Comment by devoncarew** [comment]
Ignoring publishing dry-run feedback:

```
  Package validation found the following potential issue:
  * The name of "lib/builder.dart", "builder", should match the name of the package, "shelf_router_generator".
    This helps users know what library to import.
  
  Package has 1 warning.
```

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ name: Publish
 on:
   pull_request:
     branches: [ master ]
+    types: [opened, synchronize, reopened, labeled, unlabeled]
```
Comment (kevmoo) [inline .github/workflows/publish.yaml:8]: What's this?

Comment (kevmoo) [reply .github/workflows/publish.yaml:8]: oh...I get it...

**Review by kevmoo — APPROVED** [review]

File: .github/workflows/publish.yaml (hunk 1 above)
Comment (devoncarew) [reply .github/workflows/publish.yaml:8]: That lets the publish bot react to label changes for a PR (the validation can be configured a bit via adding a `publish-ignore-warnings` label to a PR).

**Pull request merged**